exec/java-exec/src/test/java/org/apache/drill/test/QueryRowSetIterator.java

```diff
@@ -99,13 +99,6 @@ public DirectRowSet next() {
     }
   }
 
-  public void printAll() {
```

Comment:
Please either leave this, or move it to a utility class. This turns out to be a very handy way to validate or debug a unit test: add a call while working on the test to visualize the results, remove the call when done.